README.md

```diff
@@ -1 +1,76 @@
-# cypress-junit-plugin
\ No newline at end of file
+# @saucelabs/cypress-junit-plugin
+
+`@saucelabs/cypress-junit-plugin`, a plugin designed to enhance your Cypress testing workflow by generating JUnit test reports.
+
+## Features
```

Comment:
We should mention why this plugin is "better" than the mocha junit reporter for Cypress. Or at least highlight what the differences are. Devs will be wondering why they should use this over xyz.